api/.eslintrc.js

```diff
@@ -1,25 +1,26 @@
 module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    project: 'tsconfig.json',
-    sourceType: 'module',
+    tsconfigRootDir: __dirname,
+    project: './tsconfig.json',
```

Comment:
just for telling eslint "yeah don't look for tsconfig.json  at project root but just look in this folder"

=> solve this issue 'Parsing error: Cannot read file '/home/bertrand/Repos/flashimages/tsconfig.json'.eslint' in vs code